### DISCUSSION TITLE
Sudden Error When Creating Variations: Property 514 does not have any enabled

### DISCUSSION THREAD

**Discussion by babychuang** [etsy/open-api#1343] [Q&A]
We’ve been using the Etsy API to create product variations successfully until December 17th, 2024. Starting from that date, all attempts to create variations with the same data and parameters that worked before now fail with the error:
There was a problem with / : Property 514 does not have any enabled offerings.
Details:
What Changed: No changes were made to our API integration or payloads.
Error Context:
The error explicitly refers to Property 514, which we’ve been using to define variations. The properties, options, and offerings were all valid and functional prior to December 17th.
API Endpoint: POST /v3/application/listings/:listing_id/variations
Behavior: This request used to create variations without issues, but now consistently returns the error mentioned above.

Checked the API documentation for updates related to variations but found no relevant changes.

**Comment by openapi-support** [comment]
Thank you for raising this issue - it's fixed!
